tests/test_project.py

```diff
@@ -6,7 +6,7 @@ class TestProject():
 
     def test_project(self):
         proj = pipenv.project.Project()
-        assert proj.name == 'pipenv'
+        assert proj.name == 'pipenv-uoEO'
```

Comment:
Are we sure this is true for all platforms? If we run the tests in `/home/my_user/test_space/pipenv` v.s. `/home/alfred/test_space/pipenv`, won't we get different values for `proj.name`?